demos/manifest.json

```
@@ -49,5 +49,16 @@
 	"dependencies": {
             "cert-manager": true
         }
+    },
+    "px-mongo": {
+        "description": "Microservice demo that generates traffic between the Mern-K8s demo app and MongoDB.",
+        "instructions": [
+            "Load testing has been automatically launched for px-mongo",
+            "",
+            " Use the px/mongodb_data and px/http_data scripts to view the backend API and MongoDB traffic",
+            " flowing through the demo app.",
+            "",
+            " Note: The Vizier version must be later than v0.14.7."
```

```suggestion
            " Note: The Vizier version must be v0.14.8 or later."
```

nit: I think it might be confusing to see the version that is just before when the tracing is expected to work.